coil-network-ktor2/src/nonJvmCommonMain/kotlin/coil3/network/ktor2/internal/utils.nonJvmCommon.kt

```diff
@@ -14,7 +14,6 @@ internal actual suspend fun ByteReadChannel.writeTo(sink: BufferedSink) {
         val packet = readRemaining(buffer.size.toLong())
         if (packet.isEmpty) break
 
-        // TODO: Figure out how to remove 'buffer' and read directly into 'sink'.
```

Comment:
Remove this since it isn't something that we're likely to do for only Ktor 2 now that Ktor 3 has a better solution.